scripts/ci/update-github

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -ex
+
+TAG_NAME=$(git tag -l --contains HEAD)
+
+rm -rf dcos-nightly
+
+# Setup Git
+git config --global user.email "$GIT_USER@users.noreply.github.com"
+git config --global user.name "MesosphereCI Robot"
+git config credential.helper 'cache --timeout=300'
+git clone "https://$GIT_USER:$GIT_PASSWORD@github.com/mesosphere/dcos" dcos-nightly
+cd dcos-nightly
+
+if [[ $TAG_NAME ]]
+then
+  git checkout -- .
```

Comment:
This line and the next one are duplicated after the else part, I think we should [DRY](http://programmer.97things.oreilly.com/wiki/index.php/Don't_Repeat_Yourself) this up and move it before the if

Reply:
👍